### PR TITLE
IssueID:1728:esp32(m5stackcore2) supports sd card.

### PR DESCRIPTION
**Pull request by yanxiaoyong-ch**
[Detail]
none

[Verified Cases]
Build Pass: <py_engine_esp32_demo>
Test Pass: <py_engine_esp32_demo>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 3fff868039247eb9b83d9ae8a7697e393a12c8bd

**Comment by skylarCai** [comment]
auto build success,number:170

File: hardware/chip/espressif_esp32/package.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## 第一部分： 基础信息
 name: espressif_esp32                                     # <必选项> 包名称 （符合C语言变量命名规则），长度少于等于64字节
-version: master                                    # <必选项> 组件版本号
```
Comment (yong171966) [inline hardware/chip/espressif_esp32/package.yaml:3]: do not modify release vresion

File: hardware/chip/espressif_esp32/package.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ license: Apache license v2.0                        # <可选项> 源代码的
 #           指定该组件依赖的组件及版本，版本支持条件比较，支持：>=v1.0, >v1.0, ==v1.0, <=v1.0, <v1.0, v1.0
 #           未指定条件时，默认为 ==，如 v1.0 与 ==v1.0
 # depends:                                 # <可选项> 该组件依赖其他的组件，合理的依赖才能保证组件能编译、使用
-  # - libc_stub: master
+  # - libc_stub: dev_aos
```
Comment (yong171966) [inline hardware/chip/espressif_esp32/package.yaml:15]: do not modify release version

**Comment by skylarCai** [comment]
autotest success,number:1387

**Pull request closed without merging**